### ISSUE TITLE
Subscribe button doesn't work - just grays out on click. 

### ISSUE THREAD

**Issue by dalerka** [meteorclub/crater#1]
Being logged in I could not subscribe to few topics and overall the website is very slow and feels glitchy (unresponsive). 

Also, although I could successfully post a comment, then I couldn't immediately edit it and after half-minute delay just saw these errors:  
Sorry, you cannot edit this comment.
Sorry, you cannot edit this comment.
Sorry, you don't have the rights to view this page.  

p.s.: I'm using Chrome on Linux.

**Comment by queso** [comment]
The site was out of memory today and I restarted it. Let me know if it seems a little better for you now, at least with subscriptions.

**Comment by dalerka** [comment]
Nope, unfortunately, the website is still sluggish and slow. The `Subscribe` button still doesn't work and the spinner doesn't spin while waiting for a page to load. Perhaps there's some problem with CDN? this is what I get in the console:  

```
Font from origin 'https://d1477spkfkmftw.cloudfront.net' has been blocked from loading by Cross-Origin Resource Sharing policy: 
No 'Access-Control-Allow-Origin' header is present on the requested resource. 
Origin 'https://crater.io' is therefore not allowed access.
```

**Comment by dalerka** [comment]
Also got this couple times (in the console): 

```
Exception while simulating the effect of invoking 'subscribePost' 
r {message: "Match error: Expected string, got undefined", 
path: "", sanitizedError: r, errorType: "Match.Error"} 
Error: Match error: Expected string, got undefined
```

and Gravatar issues:

```
https://secure.gravatar.com/avatar/d2d2281499d028456681def3414cb92f?default=404&size=200 
Failed to load resource: the server responded with a status of 404 (Not Found)
```

**Comment by queso** [comment]
I am not sure what to do about Gravatar on that front...

As for the subscribePost issue, looks like something broken with the 0.24 release maybe? Seems like a core telescope issue. @sachag?
